Source/DafnyLanguageServer/Language/Symbols/SymbolTable.cs

```diff
@@ -62,7 +62,10 @@ bool symbolsResolved
     /// <returns><c>true</c> if a symbol was found, otherwise <c>false</c>.</returns>
     /// <exception cref="System.InvalidOperationException">Thrown if there was one more symbol at the specified position. This should never happen, unless there was an error.</exception>
     public bool TryGetSymbolAt(Position position, [NotNullWhen(true)] out ILocalizableSymbol? symbol) {
-      symbol = LookupTree.Query(position).SingleOrDefault();
+      var symbolsAtPosition = LookupTree.Query(position);
+      // TODO: Figure out why duplication crashes SingleOrDefault before using FirstOrDefault 
+      // Use case: function f(a: int) {}, and hover over a.
+      symbol = symbolsAtPosition.FirstOrDefault();
```

Comment:
Since there should only be one `LookUp` entry per designator, one designator is registered twice or overlaps with the range of another.
This should be fixed as this probably hides another error.